src/Features/CSharp/Portable/EncapsulateField/CSharpEncapsulateFieldService.cs

```diff
@@ -37,9 +37,15 @@ internal class CSharpEncapsulateFieldService : AbstractEncapsulateFieldService
             }
 
             var tempAnnotation = new SyntaxAnnotation();
-            var newIdentifier = SyntaxFactory.Identifier(originalFieldName)
-                                             .WithTrailingTrivia(declarator.Identifier.TrailingTrivia)
-                                             .WithLeadingTrivia(declarator.Identifier.LeadingTrivia);
+            var escapedName = originalFieldName.EscapeIdentifier();
```

Comment:
Why are we doing this?  Why not get the text as originally written in the declaration of the field?  that way if it uses things like ```\u``` we preserve that.